rockcraft/services/package.py

```diff
@@ -163,7 +163,11 @@ def _pack(
 
     emit.progress("Adding Pebble entrypoint")
 
-    new_image.set_entrypoint()
+    new_image.set_entrypoint(project.entrypoint_service)
+    if project.services and project.entrypoint_service in project.services:
```

Comment:
```suggestion
    if project.services:
```

The 2nd condition is redundant because if that wasn't the case, an exception would've already been raised in the project validation at https://github.com/canonical/rockcraft/pull/410/files#diff-54f38854c7e2d7d0fb5ab8160a493b4cf8f6f7da496a149eeff2680ec7ae6dbbR343

Reply:
Actually, I had to revert this requested change because the linters start complaining because of the following line if the change is made:
https://github.com/canonical/rockcraft/actions/runs/6983627927&source=gmail&ust=1700936191049000&usg=AOvVaw06UHcYXWgZrPqgXarPnzoQ